samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/BasemapProjections.razor

```diff
@@ -41,5 +41,5 @@
         return Task.CompletedTask;
     }
 
-    private SpatialReference _spatialReference = new() { Wkid = 102100 };
+    private SpatialReference _spatialReference = new(102100);
```

Comment:
Does us changing these to have constructors constitute a breaking change?

Reply:
No, you can still call the parameterless constructor, but you will get a warning (which is how it was previously).